File: solutions/javascript/lasagna/2/lasagna.js

const PREPARATION_MINUTES_PER_LAYER = 2;
export const EXPECTED_MINUTES_IN_OVEN = 40;
let actualMinutesInOven = 30;
let numberOfLayers =  2;
export function remainingMinutesInOven(actualMinutesInOven) {
 return EXPECTED_MINUTES_IN_OVEN-actualMinutesInOven;
}
export function preparationTimeInMinutes(numberOfLayers) {
  return PREPARATION_MINUTES_PER_LAYER*numberOfLayers;
}
export function totalTimeInMinutes(numberOfLayers, actualMinutesInOven) {
  let time = (numberOfLayers*PREPARATION_MINUTES_PER_LAYER)+actualMinutesInOven;
  return time;
}

File: solutions/java/lasagna/2/src/main/java/Lasagna.java
public class Lasagna {
    public int expectedMinutesInOven(){
        return 40;
    }
    public int remainingMinutesInOven(int a){
        return expectedMinutesInOven()-a;
    }
    public int preparationTimeInMinutes(int a){
        return a*2;
    }
    public int totalTimeInMinutes(int a,int b){
        return (a*2)+b;
    }
}
